requirements/edx/github.txt

```diff
@@ -28,7 +28,7 @@ git+https://github.com/mfogel/django-settings-context-processor.git
 git+https://github.com/pmitros/pyfs.git@96e1922348bfe6d99201b9512a9ed946c87b7e0b
 
 # Our libraries:
--e git+https://github.com/edx/XBlock.git@aed464a0e2f7478e93157150ac04133a745f5f46#egg=XBlock
+-e git+https://github.com/edx/XBlock.git@85d573814f9be80661d3039361f0376421ae6e98#egg=XBlock
```

Comment:
This will have to be changed once https://github.com/edx/XBlock/pull/299 is merged.